spring-cloud-dataflow-server-local/src/main/resources/dataflow-server.yml

```diff
@@ -15,7 +15,7 @@ security:
 
 spring:
   application:
-    name: spring-cloud-dataflow-server
+    name: spring-cloud-dataflow-server-local
```

Comment:
Since this yml file is now in the implementation specific I thought having a specific name which is consistent with other SPI implementations. For instance CF server has [this] ( https://github.com/spring-cloud/spring-cloud-dataflow-admin-cloudfoundry/blob/master/spring-cloud-dataflow-server-cloudfoundry/src/main/resources/dataflow-server.yml#L28)

By config server consistency, do you mean the same config properties can be shared across the SPI implementations? If so, I think we can make that change (changing the spring.application.name to `spring-cloud-dataflow-server` across all the SPI implementations)